### ISSUE TITLE
Broken links after update

### ISSUE THREAD

**Issue by EthanBlockson** [api3dao/old-api3-docs#1241]
A lot of links become unreachable (e.g. https://docs.api3.org/airnode/v0.9/reference/deployment-files/config-json.html#options-fulfillmentgaslimit)

**Comment by wkande** [comment]
As of March 2023 a newer site for the docs was released that starts with Airnode version `v0.11`. The docs you're trying to access are now referred to as the "Legacy Documentation". The link you refer to can still be found at:
https://old-docs.api3.org/airnode/v0.9/reference/deployment-files/config-json.html#options-fulfillmentgaslimit

You can find the `options.fulfillmentGasLimit` in the newer docs (Airnode `v0.11` and up) at this URL which has changed as https://docs.api3.org/reference/airnode/latest/deployment-files/config-json.html#options-fulfillmentgaslimit or https://docs.api3.org/reference/airnode/v0.11/deployment-files/config-json.html#options-fulfillmentgaslimit.


Also if you visit Airnode on the newer docs set you will see a pick list that will take you directly to the Airnode version `v0.9` you are trying to access.

To summarize:
New docs: https://docs.api3.org/
Legacy docs: https://old-docs.api3.org/


<img width="291" alt="Screenshot 2023-05-09 at 6 15 32 AM" src="https://github.com/api3dao/api3-docs/assets/3522283/a92b0452-447d-40d4-b034-0fee5bbe9a95">

**Comment by wkande** [comment]
See this [**GitHub issue**](https://github.com/api3dao/vitepress-docs/issues/467) for an additional solution to be added.

**Comment by bbenligiray** [comment]
This seems resolved

**Comment by dcroote** [comment]
Agreed given the solution implemented by @wkande
